src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -111,7 +114,7 @@ private unsafe bool InitCultureData()
                 }
                 // We found a locale name, so use it.
                 // In vista this should look like a sort name (de-DE_phoneb) or a specific culture (en-US) and be in the "pretty" form
-                this.sSpecificCulture = new String(pBuffer, 0, result - 1);
+                _sSpecificCulture = new String(pBuffer, 0, result - 1);
```

Comment:
The code formatter did not work as expected for corelib sources for some reason. I have even tried to run the latest version of the code formatter to see whether it would make a difference, but it did not.